packages/sources/src/styling/variables/zindex.scss

```diff
@@ -1,5 +1,31 @@
-$zindex-navbar: 1000;
-$zindex-dropdown: 1000;
-$zindex-popover: 1060;
-$zindex-tooltip: 1070;
-$zindex-navbar-fixed: 1030;
+// Selectors
+$zx-viewer-grid-sizing-buttons:     -1;
+$zx-viewer-grid-map-panel:          100;
+
+$zx-selectable:                     100;
+
+$zx-datepicker-base:                100;
+$zx-datepicker-dialog:              210;
+
+$zx-description-open:               100;
+
+$zx-searchbar-icon:                 101;
+
+$zx-form-control-feedback:          101;
+$zx-input-group:                    101;
+
+$zx-leaflet:                        110;
+
+// Dropdowns
+$zx-autosuggest:                    200;
+$zx-dropdown-options:               220;
+
+// Tooltip/Toggletip
+$zx-tooltip:                        410;
+
+// Modals
+$zx-nav:                     500;
```

Comment:
Wat een malle uitlijning.
Ofwel zet het allemaal op een rijtje, of niet. En hoe dan ook zo dicht mogelijk bij de langste, maar niet daar ook nog vele spaties vandaan.

Reply:
Dit uitlijnen was een idee van mij, maar om eerlijk te zijn kom ik er ook wel weer een beetje op terug 🤔 

Een nieuwe langste variable zorgt namelijk voor een change over de HELE file: ALLE waarden moeten opschuiven.

Mag ik mijn idee nog terugtrekken? 😇

Reply:
Natuurlijk trekken we dat idee terug